hw1/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="css/style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>New Jersey</title>
+</head>
+<body>
+    <div class="weather">
+        <div class="weather__top">
+            <div class="wt__item">
```

Comment:
The structure of the element name follows the scheme: **block-name**__ element-name. The name is separated from the block name of two underscores (__).
For example: 
```
<!-- Блок `search-form` -->
<form class="search-form">
    <!-- Элемент `input` блока `search-form` -->
    <input class="search-form__input">

    <!-- Элемент `button` блока `search-form` -->
    <button class="search-form__button">Найти</button>
</form>
```
Please fix the elements names in the whole file.

https://ru.bem.info/methodology/quick-start/